src/constants/Api.js

```diff
@@ -0,0 +1,18 @@
+
+
+export const GET_EXPENSE = "http://localhost:8081/tracker/register/getExpense";
+export const GET_INCOME =  "http://localhost:8081/tracker/register/getIncome";
+export const GET_INCOME_EXPENSE = "http://localhost:8081/tracker/register/getIncomeExpense";
+export const CAT_BAR_MONTH = `http://localhost:8081/tracker/register/monthlycategorysum`;
+export const CAT_BAR_YEAR = `http://localhost:8081/tracker/register/yearlycategorysum`;
+export const LINE_MONTH = `http://localhost:8081/tracker/register/TotalExpenseTotalIncomeLine_month`;
+export const LINE_YEAR = `http://localhost:8081/tracker/register/TotalExpenseTotalIncomeLine_year`;
+export const BAR_MONTH = `http://localhost:8081/tracker/register/TotalExpenseTotalIncomeBar_month`;
+export const BAR_YEAR = `http://localhost:8081/tracker/register/TotalExpenseTotalIncomeBar_year`;
+export const CAT_LIST_INCOME = 'http://localhost:8081/tracker/register/liscategoryincome';
+export const CAT_LIST_EXPENSE = 'http://localhost:8081/tracker/register/liscategoryexpense';
+export const ADD_INCOME = "http://localhost:8081/tracker/register/addincome";
+export const ADD_EXPENSE = "http://localhost:8081/tracker/register/addexpense";
+export const ADD_INCOME_CATEGORY = "http://localhost:8081/tracker/register/addincomecategory";
+export const ADD_EXPENSE_CATEGORY = "http://localhost:8081/tracker/register/addexpensecategory";
+export const REGISTER = "http://localhost:8081/tracker/register/";
```

Comment:
@Sruthyparvathy Set "http://localhost:8081/tracker/register/" this as a separate "BASE_URL" constant and remove it from all constant as it is repeating in all the constants.